pkg/apis/core/type_landscaperdeployment.go

```diff
@@ -0,0 +1,50 @@
+// SPDX-FileCopyrightText: 2021 "SAP SE or an SAP affiliate company and Gardener contributors"
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package core
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// The LandscaperDeployment is created to define a deployment of the Landscaper Service.
+type LandscaperDeployment struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// Spec contains the specification for the LandscaperDeployment
+	Spec LandscaperDeploymentSpec `json:"spec"`
+
+	// Status contains the status of the LandscaperDeployment.
+	// +optional
+	Status LandscaperDeploymentStatus `json:"status"`
+}
+
+// LandscaperDeploymentSpec contains the specification for a LandscaperDeployment.
+type LandscaperDeploymentSpec struct {
+	// Purpose contains the purpose of this LandscaperDeployment.
+	Purpose string `json:"purpose"`
+	// Region selects the region this LandscaperDeployment should be installed on.
+	// +optional
+	Region string `json:"region,omitempty"`
```

Comment:
I guess this is not needed for the first version?

Reply:
No, it not needed currently. I have added it because it may become useful in the future.
Should I remove it?

Reply:
no its ok. also when it can be omitted.
Otherwise we would need to validate that specific region is available or can be configured

Reply:
It is optional and when not specifyed, ignored by the selection of the target cluster.